### DISCUSSION TITLE
Hi, I Have a few questions and I don't think it's going to be a one answer thing.

### DISCUSSION THREAD

**Discussion by UltraWyatt** [parcoillegacy/nativegames.net-v1#7] [General]
1. Where do you get the resources (games) for nativegames.net?
2. Would you like host a second domain at a link on my website ultrawyatt.com (perhaps nativegames.ultrawyatt.com ?)
3. Also, what do you think of my website?
4. When will you have more domains?
5. Do you need help finding domains?

**Comment by thedogecraft** [comment]
> 1. Where do you get the resources (games) for nativegames.net?

> [https://nativegames.net/credits](https://nativegames.net/credits)
> 2. Would you like host a second domain at a link on my website ultrawyatt.com (perhaps nativegames.ultrawyatt.com ?)

> Yes. Please Contact on discord Dogecraft#9990

> 3. Also, what do you think of my website?

> Good but google sites is commonly blocked at schools.

> 4. When will you have more domains?

> We are actively working on adding more domains

> 5. Do you need help finding domains?

> Yes. Please contact me on discord
>
